### PR TITLE
chore: workflow changes

### PR DESCRIPTION
**Pull request by daniel-sanche**
This PR does some prep work for our git workflow updates:
- replaces references to `master` with `main`
- updates GitHub Actions to reference `main` and `develop` instead of `master`
- updated make-release.sh to remove the need to roll-back manifest changes
- added new policies to CONTRIBUTING.md (https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/issues/793)
- created new GitHub Action for re-deploying the website on each push to develop

### CODE REVIEW

File: .github/workflows/update-website.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Update Website"
+on:
+  push:
+    # run on pushes to main (after merging release branches)
+    branches:
+      - main
```
Comment (simonz130) [inline .github/workflows/update-website.yml:20]: do we want a similar action for develop branch so we always have "bleeding edge" version of the code deployed somewhere for observation?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,6 +33,24 @@ information on using pull requests.
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google.com/conduct/).
 
+## Git Workflow
+
+This project conforms to the [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-wokflow).
+The repo contains two important branches: the `main` branch stores the official release history, and the `develop` branch contains
+the current working codebase.
```
Comment (simonz130) [inline CONTRIBUTING.md:40]: nit: bullet points for branches names/descriptions for easier readability

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -33,6 +33,24 @@ information on using pull requests.
...
+
+As a contributor, your workflow will be to create a temporary branch off of `develop`, add your code, and create a PR back to `develop`.
+Periodically, we will coordinate to publish release PRs to merge the latest state of `develop` back into `main`.
```
Comment (simonz130) [inline CONTRIBUTING.md:43]: I suggest adding git commands example and put it here to describe the entire workflow. It will make new contributors' life easier

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -33,6 +33,24 @@ information on using pull requests.
...
+
+
+## Conventional Commits
+
+This project expects PR titles to conform to the [conventional commits standard](https://www.conventionalcommits.org/en/v1.0.0/).
+In brief, your PR titles should be prefixes with `feat:`, `fix:`, `chore:`, `docs:`, or `test:`, to make it easy to parse for change logs.
+A bot will enforce this as part of our PR checks.
```
Comment (simonz130) [inline CONTRIBUTING.md:50]: It's not part of this PR, right?

File: RELEASING.md [hunk 5]
```diff
@@ -5,23 +5,23 @@ There are two artifacts that make up a Stackdriver Sandbox release:
...
-   [`push-tags` CI job](https://github.com/GoogleCloudPlatform/stackdriver-sandbox/blob/master/.github/workflows/push-tags.yml).
+   [`push-tags` CI job](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/main/.github/workflows/push-tags.yml).
    Check that the job completed successfully, and the tagged images appear in the
    [`stackdriver-sandbox-230822` GCR repo](http://console.cloud.google.com/gcr/images/stackdriver-sandbox-230822)
-1. When the PR has been reviewed and thoroughly tested, merge it into master
-   - Don't squash; we must keep the tagged commit in the git history
-1. The merge to master should kick off a
-   [`github-pages` job](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/deployments/activity_log?environment=github-pages)
+1. When the PR has been reviewed and thoroughly tested, merge it into main
+   - Create a merge commit so the history is preserved in main
```
Comment (simonz130) [inline RELEASING.md:22]: command example please

File: make-release.sh [hunk 6]
```diff
@@ -71,18 +71,6 @@ else
...
 
-    # change back manifests to :latest
-    find "${REPO_ROOT}/kubernetes-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
-    find "${REPO_ROOT}/loadgenerator-manifests" -name '*.yaml' -exec sed -i -e "s/:${NEW_VERSION}/:latest/g" {} \;
-    git add "${REPO_ROOT}/kubernetes-manifests/*.yaml"
-    git add "${REPO_ROOT}/loadgenerator-manifests/*.yaml"
-
-    # change back telemetry Pub/Sub topic to "Test" topic
-    sed -i -e "s/topic_id = \"${PROD_TOPIC}\"/topic_id = \"${TEST_TOPIC}\"/g" ${REPO_ROOT}/terraform/telemetry.py;
-    git add "${REPO_ROOT}/terraform/telemetry.py"
```
Comment (simonz130) [inline make-release.sh:82]: How is this going to be handled in the "new world? Do we have a configuration that distinguishes test/prod environments?

File: .github/workflows/update-website.yml (hunk 1 above)
Comment (daniel-sanche) [reply .github/workflows/update-website.yml:20]: I ended up adding it to this PR, which I made last week: https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/785

File: CONTRIBUTING.md (hunk 4 above)
Comment (daniel-sanche) [reply CONTRIBUTING.md:50]: No, but we don't need a PR for this one, so I included the CONTRIBUTING update here

File: make-release.sh (hunk 6 above)
Comment (daniel-sanche) [reply make-release.sh:82]: In develop, we use `test` and on main, we use `prod`. I'm going to make another PR with a new GitHub Action to enforce this

File: RELEASING.md (hunk 5 above)
Comment (daniel-sanche) [reply RELEASING.md]: This is actually through the GitHub UI. I'll change the wording to make it more clear
